packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -132,6 +135,39 @@ class FlutterDriver {
 
   static int _nextDriverId = 0;
 
+  /// Connects to an Isolate of the specific ID.
+  ///
+  /// Assumes:
+  /// -- The Isolate is already running (will not wait for it to start).
+  /// -- The Isolate specified by the [isolateId] exists on the Dart VM.
+  /// -- [dartVmServiceUrl] points to a valid instance of the DartVM.
+  ///
+  /// Remaining params are identical to [FlutterDriver.connect].
+  static Future<FlutterDriver> connectToIsolate(
+    String dartVmServiceUrl, {
+    int isolateId,
```

Comment:
They'd get it from `IsolateRef`'s `id` field.

Reply:
The main thing I'm trying to avoid is bleeding dependencies into either FRDP or flutter driver.  Is there a more straightforward way to select an Isolate from an instance of the Dart VM service?

Reply:
Done!  I've also updated it to `number` since that's a more accurate reflection of the \@ Isolate implementation.